spec/controllers/api/oceans_controller_spec.rb

```diff
@@ -6,15 +6,15 @@
   let(:ocean_service) { instance_double(OceanService, call: 'ocean') }
 
   it 'renders 4 oceans' do
-    get :index
+    get :index, n_oceans: '4'
```

Comment:
that's already in the specs...   it 'renders 4 oceans, when there are no additional ```
parameters' do
    get :index
    expect(response).to be_success
    expect(oceans.size).to be 4
end
```